addons/contexts/package.json

```diff
@@ -41,7 +41,20 @@
     "qs": "*",
     "rax": "*",
     "react": "*",
-    "vue": "*"
+    "vue": "*",
```

Comment:
@gaetanmaisse this PR somehow got removed: https://github.com/storybookjs/storybook/pull/7675

Do you think I should reapply that first before this PR? Would it change this part of this PR?

Reply:
I think we can keep it as it is and as it will be with this PR, from my point of view it's more semantically correct to have `preact`, `rax` and `vue` listed as peerDep as we are importing them in the source code of `addon-context` (in `addons/contexts/preview/framework`). Adding the `peerDependenciesMeta` info will prevent any npm/yarn warning about missing dep which was the main point of https://github.com/storybookjs/storybook/issues/7625 and https://github.com/storybookjs/storybook/issues/7625#issuecomment-517627982.

Reply:
OK cool